src/scripts/Build-Toolkit.ps1

```diff
@@ -111,36 +117,27 @@ Get-ChildItem "$PSScriptRoot/../templates/$Template*" -Directory -ErrorAction Si
         }
     }
 
-    Build-MainBicep $destDir
-
-    # Copy version file last to override placeholder
-    $ver | Out-File "$destDir/modules/ftkver.txt" -NoNewline
-
-    Write-Host ''
-}
-
-# TODO: review build logic to make it more generic across all toolkit components
-# Package optimization engine
-$srcDir = "$PSScriptRoot/../optimization-engine"
-Write-Host "Building optimization engine..."
-
-# Create target directory
-$destDir = "$outdir/optimization-engine"
-Remove-Item $destDir -Recurse -ErrorAction SilentlyContinue
-& "$PSScriptRoot/New-Directory" $destDir
-
-# Copy required files
-Write-Host "  Copying files..."
-Get-ChildItem $srcDir | Copy-Item -Destination $destDir -Recurse -Exclude ".buildignore"
+    # TODO: Create a way to define required dependencies
```

Comment:
This should be implemented in a more repeatable way in the future when we understand how to best implement module dependencies. For now, I hard coded copying workbook template files and trimming the ones we don't need.